frontend/src/scenes/Workspace/routines/index.ts

```diff
@@ -0,0 +1,3 @@
+import { createRoutine } from 'redux-saga-routines';
+
+export const postWorkspaceNameRoutine = createRoutine('ADD_WORKSPACE');
```

Comment:
Better to use name the same way as you action -> addWorkspaceRoutine, for example.